docs/markdown/Machine-files.md

```diff
@@ -84,21 +121,79 @@ command line will override any options in the native file. For example, passing
 
 In addition to special data that may be specified in cross files, this
 section may contain random key value pairs accessed using the
-`meson.get_external_property()`
+`meson.get_external_property()`, or `meson.get_cross_property()`.
```

Comment:
```suggestion
`meson.get_external_property()` or `meson.get_cross_property()`.
```
And my comment about unneeded `get_cross_property` still stand, so the following would be even better:
```suggestion
`meson.get_external_property()`.
```